modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/prepare/pruning/PartitionPruningPredicateSelfTest.java

```diff
@@ -205,13 +208,15 @@ private static PartitionWithConsistencyToken computeExpectedPartition(
         return new PartitionWithConsistencyToken(p, expected.enlistmentConsistencyToken());
     }
 
-    private static List<NodeWithConsistencyToken> randomAssignments(IgniteTable table, List<String> nodeNames) {
-        return IntStream.range(0, table.partitions())
+    private static Int2ObjectMap<NodeWithConsistencyToken> randomAssignments(IgniteTable table, List<String> nodeNames) {
+        Map<Integer, NodeWithConsistencyToken> map = IntStream.range(0, table.partitions())
```

Comment:
You can create an instance of `Int2ObjectOpenHashMap` directly via old fashioned for-loop w/o creating a temporary map.

Reply:
done thanks